index.html

```diff
@@ -87,3 +86,299 @@ <h4 class="sub-title">I'm a software developer</h4>
+          <h2 class="about-me-title">About me</h2>
+          <p class="about-me-description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <a class="about-me-link" href="#">Get My Resume</a>
+          <img class="about-me-image" src="" alt="" />
+
+          <hr class="about-me-line" style="margin-top: 60%" ; />
+
+          <!-- Languages -->
+          <div class="languages">
+            <img class="lang-image" src="images/rectangle.svg" alt="" />
+            <h2 class="lang-title">Language</h2>
+          </div>
+
+          <div>
+            <ul class="about-me-list">
+              <li class="about-me-list-item">
+                <h3>Javascript</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>HTML</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>CSS</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>Ruby</h3>
+              </li>
+            </ul>
+          </div>
+
+          <!-- Framework -->
+          <div class="framework">
+            <img class="framework-image" src="images/square.svg" alt="" />
+            <h2 class="framework-title">Framework</h2>
+          </div>
+
+          <div>
+            <ul class="about-me-list">
+              <li class="about-me-list-item">
+                <h3>Bootstrap</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>Ruby on Rails</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>RSPec</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>CapyBara</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>Selenium</h3>
+              </li>
+            </ul>
+          </div>
+
+          <!-- Skills -->
+          <div class="skills">
+            <img class="skills-image" src="images/circle.svg" alt="" />
+            <h2 class="skills-title">Skills</h2>
+          </div>
+
+          <div>
+            <ul class="about-me-list">
+              <li class="about-me-list-item">
+                <h3>Codekit</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>Github</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>Codepen</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3>Gitlab</h3>
+              </li>
+
+              <li class="about-me-list-item">
+                <h3 style="margin-bottom: 100px">Terminal</h3>
+              </li>
+            </ul>
+          </div>
```

Comment:
- [ ] Please you should make sure that your design stick as much as possible to the Figma design, that means (font, margin, padding, color,...). You should make sure that the space between your languages, skills, and framework is the same as in the Figma, also the fonts 🙃 😉 

Your design

<img width="266" alt="Screen Shot 2022-10-25 at 20 43 56" src="https://user-images.githubusercontent.com/74501523/197864659-48e1cd6b-62f0-40b1-ae70-0a89481bfdf1.png">

Figma design

<img width="266" alt="Screen Shot 2022-10-25 at 20 44 22" src="https://user-images.githubusercontent.com/74501523/197864729-b03683a3-8423-40d1-9e6c-96a8a21f709c.png">

Reply:
Please I have resolved these. Kindly check again

Reply:
Fixed